### PR TITLE
Adapt test-image-delete-from-secondary to test for both ecpools and replicated pools

### PR DESCRIPTION
**Pull request by rkhambad**
Signed-off-by: Rajendra Khambadkar <rkhambad@rkhambad.remote.csb>

Goal is to verify test case execution on both Replication pool as well as on EC pool.

success log: 
6.0 : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-vor9d/

### CODE REVIEW

File: tests/rbd_mirror/test-image-delete-from-secondary.py [hunk 1]
```diff
@@ -1,58 +1,20 @@
...
-            imagename=imagename,
-            imagesize=config.get("imagesize", "1G"),
-            io_total=config.get("io-total", "1G"),
-            mode="image",
-            mirrormode="snapshot",
-            **kw,
-        )
+        pool = config[pool_type]["pool"]
+        image = config[pool_type]["image"]
+        imagespec = pool + "/" + image
```
Comment (sunilangadi2) [inline tests/rbd_mirror/test-image-delete-from-secondary.py:17]: add success return code, check the logs it's failing for 
`2023-02-02 11:45:35,212 (cephci.test-image-delete-from-secondary) [ERROR] - cephci.ceph.ceph.py:1577 - rbd: error opening image ec_imagenccscwMakd: (2) No such file or directory`
I think it's expected Is it?

File: tests/rbd_mirror/test-image-delete-from-secondary.py [hunk 2]
```diff
@@ -1,58 +1,20 @@
 import datetime
 import time
```
Comment (sunilangadi2) [inline tests/rbd_mirror/test-image-delete-from-secondary.py:2]: please removed unused imports? `time` and `datetime` no where used in this module.

**Review by sunilangadi2 — CHANGES_REQUESTED** [review]
Please add test results from 5.x pacific as well

File: tests/rbd_mirror/test-image-delete-from-secondary.py (hunk 1 above)
Comment (rkhambad) [reply tests/rbd_mirror/test-image-delete-from-secondary.py:17]: it is expected to fail.

File: tests/rbd_mirror/test-image-delete-from-secondary.py (hunk 2 above)
Comment (rkhambad) [reply tests/rbd_mirror/test-image-delete-from-secondary.py:2]: time and datetime is used in line no 22 and 54

**Review by sunilangadi2 — APPROVED** [review]
LGTM

**Comment by rkhambad** [comment]
success Logs for 5.3 : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-3b4du/

**Review by VasishtaShastry — APPROVED** [review]

**Review by manasagowri — APPROVED** [review]
Group review done.

**Pull request merged**